tracks/r/exercises/rotational-cipher/mentoring.md

```diff
@@ -0,0 +1,34 @@
+### Reasonable solution
+
+This solution is clear, robust, and extremely concise.
+
+```r
+convert_ascii <- function(value, key){
+  if ((value >= 97) && (value <= 122)){
```

Comment:
Extract magic numbers to constants. The reasonable solution should be as optimal as possible; you can then decide if having magic numbers in the code is approvable or not.

Reply:
Should I move the "suggestion" to remove magic numbers to a "talking point" and then have a solution without any magic numbers?

Seems like plenty of solutions slip into the community solutions while using magic numbers.

Reply:
Yep, unfortunately. Let's get this merged and fix in another iteration. The suggestion is _valid_! I would just like it to be applied to the reasonable solution as mentors tend to put that solution on a pedestal and talk from its point of view ;)